packages/pymedphys/src/pymedphys/deliverydata.py

```diff
@@ -1,4 +1,25 @@
-from pymedphys_deliverydata.object import DeliveryData
-from pymedphys_deliverydata.dicom import (
-    delivery_data_from_dicom,
-    delivery_data_to_dicom)
+from pymedphys_base.deliverydata import DeliveryData as DeliveryDataBase
+
+to_be_combined = [DeliveryDataBase]
+
+try:
+    from pymedphys_dicom.deliverydata import DeliveryDataDicom
+    to_be_combined += [DeliveryDataDicom]
+except ImportError:
+    pass
+
+try:
+    from pymedphys_fileformats.deliverydata import DeliveryDataLogfile
```

Comment:
Perhaps not something to worry about yet, but would this need further stratification due to multiple log file vendors and major versions? E.g.
- `ElektaLog`
- `VarianTrajectoryLog`
- `VarianDynalog`

Reply:
Very possibly. This is part of our internal API however, so we can just change that when it is needed.

Reply:
No worries